test/tests/treeview-1a.js

```diff
@@ -0,0 +1,675 @@
+'use strict';
+
+const { ariaTest } = require('..');
+const { By, Key } = require('selenium-webdriver');
+const assertAttributeValues = require('../util/assertAttributeValues');
+const assertAriaLabelledby = require('../util/assertAriaLabelledby');
+const assertRovingTabindex = require('../util/assertRovingTabindex');
+
+const exampleFile = 'treeview/treeview-1/treeview-1a.html';
+
+const ex = {
+  treeSelector: '#ex1 [role="tree"]',
+  treeitemSelector: '#ex1 [role="treeitem"]',
+  groupSelector: '#ex1 [role="group"]',
+  folderSelector: '#ex1 [role="treeitem"][aria-expanded]',
+  topLevelFolderSelector: '#ex1 [role="tree"] > [role="treeitem"]',
+  nextLevelFolderSelector: '[role="group"] > [role="treeitem"][aria-expanded]',
+  docSelector: '#ex1 .doc[role="treeitem"]',
+  textboxSelector: '#ex1 #last_action'
+};
+
+const openAllFolders = async function (t) {
+  const closedFoldersSelector = ex.treeitemSelector + '[aria-expanded="false"]';
+  let folders = await t.context.session.findElements(By.css(closedFoldersSelector));
+
+  // While their are closed folders
+  while (folders.length != 0) {
+
+    // Find the first visible closed folder and click it
+    for (let folder of folders) {
+      if (await folder.isDisplayed()) {
+        await folder.click();
+        break;
+      }
+    }
+
+    folders = await t.context.session.findElements(By.css(closedFoldersSelector));
+  }
+};
+
+const checkFocus = async function (t, selector, index) {
+  return await t.context.session.executeScript(function (/* selector, index*/) {
+    const [selector, index] = arguments;
+    const items = document.querySelectorAll(selector);
+    return items[index] === document.activeElement;
+  }, selector, index);
+};
+
+const checkFocusOnParentFolder = async function (t, el) {
+  return await t.context.session.executeScript(function () {
+    const el = arguments[0];
+
+    // the element is a folder
+    if (el.hasAttribute('aria-expanded')) {
+      return document.activeElement == el.parentElement.closest('[role="treeitem"][aria-expanded]');
```

Comment:
`===` strict equals everywhere so we don't inadvertently introduce truthy comparisons of existence rather than reference equivalence.